src/components/v1/DepositAndBridgeButton.tsx

```diff
@@ -0,0 +1,322 @@
+                    placeholder="0.00"
+                    value={depositAmount}
+                    onChange={(e) => setDepositAmount(e.target.value)}
+                    type="number"
+                    {...inputProps}
+                  />
+                  <InputRightElement width="4.5rem">
+                    <Button
+                      h="1.75rem"
+                      size="sm"
+                      onClick={() => setDepositAmount(balance.toString())}
+                    >
+                      MAX
+                    </Button>
+                  </InputRightElement>
+                </InputGroup>
+                <FormHelperText>
+                  Balance: {balance.toFixed(6)} {selectedToken?.displayName}
+                </FormHelperText>
+              </FormControl>
+
+              {/* Minimum Mint */}
+              <FormControl>
+                <FormLabel>Minimum Shares to Mint</FormLabel>
+                <Input
+                  placeholder="0.00"
+                  value={minimumMint}
+                  onChange={(e) => setMinimumMint(e.target.value)}
+                  type="number"
+                  {...inputProps}
+                />
+                <FormHelperText>
+                  Slippage protection - minimum shares you'll accept
+                </FormHelperText>
+              </FormControl>
+
+              {/* Destination Chain */}
+              <FormControl>
+                <FormLabel>Destination Chain</FormLabel>
+                <Select
+                  value={destinationChain}
+                  onChange={(e) => setDestinationChain(e.target.value as LayerZeroChain)}
+                >
+                  {availableChains.map((chain) => (
+                    <option key={chain} value={chain}>
+                      {getChainDisplayName(chain)}
+                    </option>
+                  ))}
+                </Select>
+              </FormControl>
+
+
+              {/* Max Fee */}
+              <FormControl>
+                <FormLabel>Max Bridge Fee (ETH)</FormLabel>
+                <Input
+                  placeholder="0.01"
+                  value={maxFee}
+                  onChange={(e) => setMaxFee(e.target.value)}
+                  type="number"
+                  step="0.001"
+                  {...inputProps}
+                />
+                <FormHelperText>
+                  Maximum fee willing to pay for bridging.
+                </FormHelperText>
+              </FormControl>
+
+              {bottomText && (
+                <Text fontSize="sm" color="gray.500" textAlign="center">
+                  {bottomText}
+                </Text>
+              )}
+            </VStack>
+          </ModalBody>
+
+          <ModalFooter>
+            <Button
+              colorScheme="blue"
+              onClick={handleDepositAndBridge}
+              isDisabled={!isValidAmount || !isValidMinimumMint || !isValidFee || depositAndBridgeStatus.loading}
```

Comment:
Nit-pick: extract this into a constant for easier debugging


```suggestion
  const isButtonDisabled = !isValidAmount || !isValidMinimumMint || !isValidFee || depositAndBridgeStatus.loading
              isDisabled={isButtonDisabled}
```